src/wallet/wallet2.h

```diff
@@ -178,6 +178,13 @@ namespace tools
       uint64_t m_timestamp;
     };
 
+    struct address_tx : payment_details
+    {
+      uint64_t m_coinbase;
+      uint64_t m_mempool;
```

Comment:
uint64_t, not bool ? I don't see where that's used in this patch so it must be in another so I can't easily check usage, but it seems odd.